qgis_deployment_toolbelt/jobs/job_environment_variables.py

```diff
@@ -24,19 +25,42 @@
     check_var_can_be_path,
 )
 from qgis_deployment_toolbelt.utils.url_helpers import check_str_is_url
-from qgis_deployment_toolbelt.utils.win32utils import (
-    delete_environment_variable,
-    refresh_environment,
-    set_environment_variable,
-)
 
 # #############################################################################
 # ########## Globals ###############
 # ##################################
 
+
 # logs
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s||%(levelname)s||%(module)s||%(lineno)d||%(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
 logger = logging.getLogger(__name__)
 
+
+# #############################################################################
+# ###### Conditional imports #######
+# ##################################
+
+
+if opersys == "linux":
+    from qgis_deployment_toolbelt.utils.linux_utils import (
+        delete_environment_variable,
+        set_environment_variable,
+    )
+elif opersys == "win32":
+    from qgis_deployment_toolbelt.utils.win32utils import (
+        delete_environment_variable,
+        refresh_environment,
+        set_environment_variable,
+    )
+else:
+    logger.debug("Unsupported operating system.")
+    exit()
```

Comment:
Mac (darwin) is missing!

And the log should be critical or error at least:

```suggestion
else:
    logger.error("Unsupported operating system.")
    exit()
```